sdk/storage/storage-blob/samples/javascript/advanced.js

```diff
@@ -15,19 +15,23 @@ const {
   ContainerURL,
   ServiceURL,
   StorageURL
-} = require(".."); // Change to "@azure/storage-blob" in your package
+} = require("../.."); // Change to "@azure/storage-blob" in your package
```

Comment:
Also mention in readme about making this change.

Reply:
**Readme imports the package "@azure/storage-blob" instead of using source-code.**

![image](https://user-images.githubusercontent.com/10452642/56157790-4716ce00-5f75-11e9-98c4-50e17fab6bd6.png)

Can you let me know on what you want me to update ?

Reply:
If I am a user, what do I need to do in order to run this sample? I think it's good to capture what I need to do in the README. I see mentioning of 'Copy the sample to your samples folder and use `node` to run it.' Will that work without the changes here?

Reply:
Oh, you mean in the samples-readme!
Got your point now. I'll update the readme for samples.